.changeset/spicy-news-lick.md

```diff
@@ -0,0 +1,9 @@
+---
+"@octopusdeploy/add-changeset": patch
+"@octopusdeploy/current-branch-name": patch
+"@octopusdeploy/extract-package-details": patch
+"@octopusdeploy/find-and-replace-all": patch
+"@octopusdeploy/shared-action-utils": patch
+---
+
+Security update for form-data
```

Comment:
This is what will end up in the release notes, I think it would be good to provide more detail about what the update was, maybe the version number and which security alert was being fixed?